.github/workflows/ci.yml

```diff
@@ -129,3 +144,87 @@ jobs:
           labels: ${{ steps.meta.outputs.labels }}
           cache-from: ${{ steps.registry_refs.outputs.tags }},mode=max
           cache-to: ${{ steps.registry_refs.outputs.tags }},mode=max
+
+  integration-tests:
+    needs: [determine-changes, lint, test, build-container]
+    runs-on: ubuntu-latest
+    if: ${{ needs.determine-changes.outputs.non-test-changes }}
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v4
+
+    - name: Extract Branch Name
+      run: |
+        if [ "$GITHUB_HEAD_REF" != "" ]; then
+          echo "BRANCH_NAME=$GITHUB_HEAD_REF" >> $GITHUB_ENV
```

Comment:
Is there a set output command?

Reply:
> set output command
Oh like pull the docker image name instead from a previous job? I could do that but this seemed less complex